modules/createGame.js

```diff
@@ -0,0 +1,17 @@
+const createGame = async () => {
+  const gameId = JSON.parse(window.localStorage.getItem('gameID')) || [];
+  if (gameId.length === 0) {
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
```

Comment:
You did a great job obviously 👍  Normally we don't expect a game creation in the code but you implemented that, that's cool. User can create their game with Postman or any other tool and save that API key in a variable. The issue occurs when you clone the repo and when localStorage is empty. The first score you sent does not go anywhere. You can solve it in two ways, when you create a game you can await, then submit the first score. The other way, is to remove the game-creating functionality and keep your API key in the repo in constant.
![Screen Recording 2023-03-15 at 8 54 57 PM](https://user-images.githubusercontent.com/25887253/225404632-a529b2b8-ac82-43cc-ac4c-4bd88a41dcea.gif)